internal/modules/modules.go

```diff
@@ -4,6 +4,7 @@ package modules
 import (
 	"context"
 	"fmt"
+	"github.com/DIMO-Network/dis/internal/modules/compass"
 
 	"github.com/DIMO-Network/dis/internal/modules/autopi"
```

Comment:
move this out of standard lib imports
```suggestion

	"github.com/DIMO-Network/dis/internal/modules/compass"
	"github.com/DIMO-Network/dis/internal/modules/autopi"
```